packages/contract-schema/test/solc.js

```diff
@@ -4,20 +4,21 @@ const Schema = require("../");
 const debug = require("debug")("test:solc");
 
 describe("solc", function () {
-  const exampleSolidity = `pragma solidity ^0.6.0;
+  const exampleSolidity = `// SPDX-License-Identifier: MIT
+  pragma solidity ^0.8.0;
```

Comment:
Worth noting that adding in those extra spaces will mean the lines of the actual Solidity source will begin with those extra spaces. :P  That's presumably why they weren't there before.  However since this has no actual effect, it's not a problem to change it.

Reply:
Thanks for explaining about the spaces @haltman-at .